backend/src/routes/index.ts

```diff
@@ -236,6 +264,8 @@ const routes: FastifyPluginAsync = async (server) => {
     "/whitelist",
     async function (request, reply) {
       try {
+        server.log.info(request.query, "query passed: ");
```

Comment:
Can't we write this as a utility function that gets called at the beginning of every function? You can dump/log whatever information you need there!